app/code/Stanislavz/Chat/view/frontend/templates/chat/chat-form.phtml

```diff
@@ -0,0 +1,48 @@
+<?php /** @var \Stanislavz\Chat\Block\ChatForm $block */ ?>
+<div class="chat-wrapper"
+     id="chat-wrapper"
+     data-mage-init='{"chatModal":{}}'
+     style="display: none"
+>
+    <div id="message-wrapper">
+        <ul class="message-list"></ul>
+    </div>
+    <form action="javascript:void(0)"
+          method="post"
+          class="chat-form"
+          id="chat-form"
+          data-mage-init='{
+            "validate": {},
```

Comment:
@ahrrhy This fails
require.js:1895 GET https://stanislav-zhuravel.local/static/version1586765617/frontend/Stanislavz/ptashka/en_US/validate.js net::ERR_ABORTED 404